session_0/item.rb

```diff
@@ -8,7 +10,7 @@ class Item
 
   # TODO: Store quantity
   attr_accessor :name, :price, :category_id, :discount_rate,
-    :discount_deadline
+    :discount_deadline, :quantity, :current_price
```

Comment:
Do you need to add `current_price` as an attribute?

Reply:
No, but i thought it would be better to mention that what all values this class holds.

Reply:
That would be wrong. To see why, imagine a line like `item.current_price = 100`. (This code works because Ruby is lax and encouraging).

Now, we know that current price is dependent on the price, discount deadline and current time - assigning a value directly defeats the purpose

Reply:
So till the time, we calculate it when the function is called rather than returning the previously stored value, It shd be fine, R8?

Reply:
I have updated the code :+1: